docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -23,61 +27,51 @@ VMs might fail to start because no boot source is available for cloning.
 
 ## Diagnosis
 
-1. Check the cluster for a default storage class:
+1. Check the cluster for a default (Kubernetes and/or virtualization) storage class:
 
    ```bash
-   $ kubectl get sc
+   $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubernetes\.io\/is-default-class=="true")].metadata.name}'
+
+   $ kubectl get sc -o jsonpath='{.items[?(.metadata.annotations.storageclass\.kubevirt\.io\/is-default-virt-class=="true")].metadata.name}'
    ```
```

Comment:
@ousleyp mentioned that in the docs, we only put one command in each code block so that they can be copied more accurately by clicking a "copy" button.

Reply:
Fixed